<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Switch_Board</class>
 <widget class="QMainWindow" name="Switch_Board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switch_Board</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ertyuikl&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::MarkdownText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_SendCmd">
       <property name="text">
        <string>Send user cmd(Hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_SendCmd">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;byte0 byte1表示所有数据的长度，&lt;/p&gt;&lt;p&gt;byte2 byte3表示命令&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_Find">
       <property name="text">
        <string>Find Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_DeviceIndex">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OpenDevice">
       <property name="text">
        <string>Open Device</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER 0 ZCUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER 2 D-T22</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DEBUG_0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>DEBUG 0 D-T22</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DEBUG_1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>DEBUG 1 ZCUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DEBUG_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>DEBUG 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DEBUG_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>DEBUG 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DEBUG_NONE">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DEBUG NONE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_ON">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POWER_OFF">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDC3C7; color: black; border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>POWER OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
